tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
 ```
 
 This will run all metrics tests and generete a `results` directory.
 
 Each file in the `result` directory contains the result for a test and has the format of a CSV.
```

Comment:
nitpick not directly related to your patches (but shows up in view)... here `result` should also be `results` as per above. If you do re-work this file then maybe you can fix that as well? :-)